### ISSUE TITLE
Equivalent for the require-jsdoc eslint rule set

### ISSUE THREAD

**Issue by vpmedia** [gajus/eslint-plugin-jsdoc#1032]
## Motivation

Report every missing piece of jsdoc comment(s).

## Current behavior

Missing class and method comments are not reported as missing.

## Desired behavior

Show warning for all missing jsdoc comments.

## Alternatives considered

If I use the following (deprecated) eslint rule set I get warnings for missing class and method declarations:

```
    "require-jsdoc": ["warn", {
      "require": {
          "FunctionDeclaration": true,
          "MethodDefinition": true,
          "ClassDeclaration": true,
          "ArrowFunctionExpression": true,
          "FunctionExpression": true
      }
    }],
```

**Comment by vpmedia** [comment]
Sorry, "jsdoc/require-description": ["warn", {"contexts":["any"]}] does what I've described.

**Comment by vpmedia** [comment]
Re-opened, I can't configure the plugin to report missing class method jsdoc comments.

**Comment by brettz9** [comment]
Btw, for the other equivalents, e.g., `valid-jsdoc`, you can see https://github.com/gajus/eslint-plugin-jsdoc/wiki/Comparison-with-deprecated-JSdoc-related-ESLint-rules

**Comment by vpmedia** [comment]
@brettz9 Thank you, this is really helpful!
